Remove outdated person if its affiliations was updated (#456)
qbicsoftware/offer-manager-2-portlet: CHANGELOG.rst

* Fix fail based on double clicking a customer in the SelectCustomerView for in the offer creation process (`#452 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/452>`_)

> * Updating a customer now removes the old customer from the customerResourceService and in the grids(`#456 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/456>`_)

Reviewer: ```suggestion
* Updating a person removes the old entry also from the customerResourceService and projectManagerResourceService (`#456 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/456>`_)
```
